src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -43,9 +44,14 @@ class Parser:
 
     def parse_request(self, parse_params: HttpRequest) -> dict:
         if Configuration.verbose and parse_params and not should_scrub_domain(parse_params.host):
+            parse_params.omit_skip_path = (
+                None if should_scrub_known_services() else self.get_omit_skip_path()
```

Comment:
as written before, this `if` should be inside the `lumigo_dumps` function and not here.